consensus-types/types/payload_test.go

```diff
@@ -232,13 +242,9 @@ func TestExecutionPayload_ToHeader(t *testing.T) {
 	require.Equal(t, payload.GetBlockHash(), header.GetBlockHash())
 	require.Equal(t, payload.GetBlobGasUsed(), header.GetBlobGasUsed())
 	require.Equal(t, payload.GetExcessBlobGas(), header.GetExcessBlobGas())
+	require.Equal(t, payload.Version(), header.Version())
```

Comment:
added the version check, since I explicitly set the payload header version from the payload one